### PR TITLE
removing unused rabbit queues from collection exercise

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# What and why?

# How to test?

# Trello
https://trello.com/c/svAgAyV2/733-remove-unused-rabbit-queue-rm-collection-exercise-service

### CODE REVIEW

File: src/main/resources/springintegration/flows.xml [hunk 1]
```diff
@@ -7,11 +7,6 @@
   http://www.springframework.org/schema/integration http://www.springframework.org/schema/integration/spring-integration.xsd">
 
     <int:annotation-config/>
-	<import resource="case-outbound-flow.xml"/>
-    <import resource="collex-event-messager-outbound.xml"/>
-    <import resource="collex-event-messager-inbound.xml"/>
```
Comment (warrenbailey) [inline src/main/resources/springintegration/flows.xml:12]: are we able to delete these files as well as removing them from here?

**Review by warrenbailey — CHANGES_REQUESTED** [review]

**Comment by warrenbailey** [comment]
looks like you need to run `mvn fmt:format`

File: src/main/resources/springintegration/flows.xml (hunk 1 above)
Comment (creativeappdevamit) [reply src/main/resources/springintegration/flows.xml:12]: yes.. don't know why my last commit didn't fo in.. I did remove it but did not check in.

Comment (creativeappdevamit) [reply src/main/resources/springintegration/flows.xml:12]: done

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**